src/chrome/content/rules/DF-SRV.xml

```diff
@@ -0,0 +1,75 @@
+<!--
+	Cert mismatch:
+		- java.cdp.df-srv.de
+		- monitor.cdp.df-srv.de
+		- py.cdp.df-srv.de
+		- sso.cdp.df-srv.de
+		- report.cdp.df-srv.de
+		- a-dev.traefik.df-srv.de
+		- crowd.traefik.df-srv.de
+		- hb.traefik.df-srv.de
+		- kue.traefik.df-srv.de
+		- r-dev.traefik.df-srv.de
+		- ro.traefik.df-srv.de
+		- sentry.traefik.df-srv.de
+		- solr.traefik.df-srv.de
+		- solr-dev.traefik.df-srv.de
+		- swarm.traefik.df-srv.de
```

Comment:
Wildcard DNS

Reply:
We should extend Sublist3r to optionally filter these out.